tests/keras/layers/test_advanced_activations.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from numpy.testing import assert_allclose
+import numpy as np
+from keras import backend as K
+
+np.random.seed(1337)
+
+def get_standard_values():
+    '''
+    These are just a set of floats used for testing the activation
+    functions, and are useful in multiple tests.
+
+    The values should all be non-negative becasue they and their negative
+    are used to test the ReLU derivates in this file.
+    '''
+    return np.array([[0, 0.1, 0.5, 0.9, 1.0, 10, 1e3, 0.0001]], dtype=K.floatx())
+
+class TestAdvancedActivations(unittest.TestCase):
+    def test_leaky_relu(self):
+        from keras.layers.advanced_activations import LeakyReLU
+        inp = get_standard_values()
+        for alpha in [0., .5, -1.]:
+            layer = LeakyReLU(alpha=alpha)
+            layer.input = K.variable(inp)
+            for train in [True, False]:
+                outp = K.eval(layer.get_output(train))
+                assert_allclose(outp,inp)
+
+            layer.input = K.variable(-inp)
+            for train in [True, False]:
+                outp = K.eval(layer.get_output(train))
+                assert_allclose(outp,-inp*alpha)
+
+            config = layer.get_config()
+            assert config['alpha'] == alpha
+
+    def test_prelu(self):
+        from keras.layers.advanced_activations import PReLU
```

Comment:
Thats a good point! In practice you'll probably get bunch of other tests failing if some import is broken though. But yes, I see the benefit in importing locally